test/addTarget.js

```diff
@@ -121,6 +121,136 @@ exports.addTarget = {
         var phases = proj.pbxCopyfilesBuildPhaseObj(target.uuid);
         test.ok(!phases);
 
+        test.done();
+    },
+    'should have \'wrapper.application\' filetype for application product': function (test) {
```

Comment:
As a minor comment, I would do something like this on all of the new cases:

```suggestion
    'should have `wrapper.application` filetype for application product': function (test) {
```

or a little less radically:

```suggestion
    "should have 'wrapper.application' filetype for application product": function (test) {
```

What do you think?

Reply:
I'll update to:
```js
'should have "xx" filetype for yy product'
```
This helps the filetype stand out, and the double quoted string is the actual expected result, so it seems to make the most sense.

Took my initial example from `addBuildPhase.js`--sorry about that.